Code/Tools/SceneAPI/SceneCore/Containers/GraphObjectProxy.cpp

```diff
@@ -105,7 +105,9 @@ namespace AZ
                 if (!type.empty())
                 {
                     AzFramework::StringFunc::Append(bufferArg, ": ");
+		    AZ_PUSH_DISABLE_WARNING_GCC("-Wdangling-pointer");
                     AzFramework::StringFunc::Append(bufferArg, type.data());
+		    AZ_POP_DISABLE_WARNING_GCC;
```

Comment:
This line doesn't produce a dangling refefence.
It is actually the call to FetchPythonType above.
That function returns an [AZStd::string](https://github.com/o3de/o3de/blob/75bbcc1ae57670ca5e2042e8a6d10cd26fc9e07d/Code/Tools/SceneAPI/SceneCore/Containers/GraphObjectProxy.cpp#L62). So the variable it is being assigned to should be an `AZStd::string`

```c++
AZStd::string_view type = FetchPythonType(*behaviorMethod.GetArgument(argIndex))
```